src/Monolog/Handler/MongoDBHandler.php

```diff
@@ -28,14 +28,20 @@
 class MongoDBHandler extends AbstractProcessingHandler
 {
     protected $mongoCollection;
+    protected $namespace;
+    protected $manager;
```

Comment:
I'd suggest making all three of these private. If needed, they can be opened up later.